.github/workflows/twister.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
     runs-on:
       group: zephyr-runner-v2-linux-x64-4xlarge
     container:
-      image: ghcr.io/zephyrproject-rtos/ci-repo-cache:v0.26.13.20240601
+      image: ghcr.io/zephyrproject-rtos/ci:main-amd64
```

Comment:
This can cause frequent local Docker image cache flushes and lead to CI performance degradation.

If all the Rust components are ready on the docker-image side, I can look into tagging a new image along with the "repo cache."

Reply:
I think the idea was to use this mainly to test that it is ready. I can update this in the branch when the tag is ready. There should be a lot of changes on the collab-rust branch.

Reply:
@stephanosio lets tag the docker, verify it here and change all other workflows on main once verified.

Reply:
Docker is tagged, and I've updated this to use that version.